tagsets/query_modules.php

```diff
@@ -445,6 +449,14 @@ function query__get_query_array($posted_array,$experiment_id="") {
                             }
                     }
                 }
+                $int_fields=participant__get_internal_freetext_search_fields();
+                foreach ($int_fields as $ifield) {
+                    if ($params['search_field']=='all') {
```

Comment:
Something happened with indentation in this block. line 445 is shifted by 8 symbols instead of 4., `elseif` should be at same level with `if`

Reply:
Ok.